src/pages/Volunteers.jsx

```diff
@@ -3,6 +3,7 @@ import GeneralInfo from '../components/GeneralInfo';
 import WeeklyInfo from '../components/WeeklyInfo';
 import RolesAndSkills from '../components/RolesAndSkills';
 import DuiAndCrimHis from '../components/DuiAndCrimHis';
+import TrainingAndForms from '../components/TrainingAndForms';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../components/TrainingAndForms"